server/gubled.go

```diff
@@ -170,6 +171,25 @@ var CreateModules = func(router router.Router) []interface{} {
 		logger.Info("APNS: disabled")
 	}
 
+	if *Config.SMS.Enabled {
+		logger.Info("Optivo SMS: enabled")
+		if *Config.SMS.APIKey == "" || *Config.SMS.APISecret == "" {
+			logger.Panic("The API Key has to be provided when NEXMO SMS connector is enabled")
```

Comment:
Remove Nexmo from log message. We should make guble separated from provider as much as possible.